source/common/json/config_schemas.cc

```diff
@@ -1016,6 +1016,40 @@ const std::string Json::Schema::CLUSTER_MANAGER_SCHEMA(R"EOF(
   }
   )EOF");
 
+const std::string Json::Schema::LDS_SCHEMA(R"EOF(
+  {
+    "$schema": "http://json-schema.org/schema#",
+    "type" : "object",
+    "properties" : {
+      "listeners" : {
+        "type" : "array",
+        "items" : {"type" : "object"}
```

Comment:
This schema doesn't imply that we're getting back valid Listener objects? Is this because we defer the schema validation to `addOrUpdateLisener()`? That seems reasonable but has slightly weaker validation capabilities of the pure schema. That said, I don't mind anyway as this is going to be legacy soon when we move to v2 LDS.